tools/packaging/deploy/enclave-cc-deploy.sh

```diff
@@ -57,6 +57,7 @@ function install_artifacts() {
 
 	cp shim-rune-config.toml /etc/enclave-cc/config.toml
 
+	mkdir -p /opt/confidential-containers/bin
 	install -D -m0755 ${shim_rune_binary} /opt/confidential-containers/bin/
```

Comment:
I actually would make sure we have this line as:
```
install -D -m0755 ${shim_rune_binary} /opt/confidential-containers/bin/${shim_rune_binary}
```

That will do the trick without the need of adding the `mkdir`, as ...
```
-D
    create all leading components of DEST except the last, then copy SOURCE to DEST
```

Reply:
+1